src/cryptography/hazmat/backends/openssl/backend.py

```diff
@@ -1461,6 +1461,17 @@ def _tmp_bn_ctx(self):
         finally:
             self._lib.BN_CTX_end(bn_ctx)
 
+    @contextmanager
+    def _tmp_bn_mont_ctx(self, modulus):
```

Comment:
Since we don't use this in cryptography this will cause a failure in CI since it is uncovered code. This probably belongs in a third party library unless/until we have a need for BN with montgomery multiplication for a primitive in cryptography.